### PR TITLE
Add Go projects for hacktoberfest 2022

### PR DESCRIPTION
**Pull request by omissis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,20 @@ Scritti in Rust:
 - Rust-av, https://github.com/rust-av, multimedia in rust
 - rav1e, https://github.com/xiph/rav1e, un encoder AV1 scritto in rust (ed assembly)
 
+Scritti in Go:
+
+- [Go Hacktoberfest](https://github.com/topics/hacktoberfest?l=go), list of Go projects tagged with "hacktoberfest"
+- [go-jsonschema](https://github.com/omissis/go-jsonschema), a tool to generate Go data types from JSON Schema definitions.
+- [goarkitect](https://github.com/omissis/goarkitect), describe and check architectural constraints of a Go project using a composable set of rules described in one or multiple yaml files.
+- [spago](https://github.com/nlpodyssey/spago), self-contained Machine Learning and Natural Language Processing library in Go.
+- [cybertron](https://github.com/nlpodyssey/cybertron), simple and easy-to-use interface for cutting-edge Natural Language Processing (NLP) technologies.
```
Comment (xrmx) [inline README.md:44]: These are not tagged as hacktoberfest aFAICS, care to poke Matteo?

File: README.md [hunk 2]
```diff
@@ -35,6 +35,20 @@ Scritti in Rust:
...
+- [capsule](https://github.com/clastix/capsule), multi-tenancy and policy-based framework for Kubernetes.
+- [kamaji](https://github.com/clastix/kamaji), build and operate Kubernetes at scale with a fraction of operational burden.
+- [inspektor gadget](https://github.com/kinvolk/inspektor-gadget), introspecting and debugging Kubernetes applications using eBPF "gadgets".
+- [permission manager](https://github.com/sighupio/permission-manager), permission Manager is a project that brings sanity to Kubernetes RBAC and Users management, Web UI FTW.
```
Comment (xrmx) [inline README.md:48]: Same as before

File: README.md (hunk 1 above)
Comment (omissis) [reply README.md:44]: all the authors have been contacted a few days ago, they all agreed to do that

Comment (xrmx) [reply README.md:44]: :rocket:

**Pull request merged**